scripts/update/update

```diff
@@ -87,10 +100,17 @@ EOF
 
 # Download new release
 if [[ "${update_type}" == "ota" ]]; then
+  # Check that release url is valid
+  if ! curl_check "${RELEASE_URL}"; then
```

Comment:
Is this downloading the update and piping it to `/dev/null` to check the response code, then downloading it again later to extract?

Reply:
Couldn't we just handle this in one go with:
```bash
if ! curl -L "${RELEASE_URL}" | tar -xz --strip-components=1; then
  # handle error
fi
```

Reply:
> Is this downloading the update and piping it to /dev/null to check the response code, then downloading it again later to extract?

`curl_check` just gets the headers (`/dev/null` is to throw away all the header boilerplate and `-w "%{http_code}"` is to get just the last response code)

> Couldn't we just handle this in one go

Oh yea this totally works as well (tested to double-check) and is much cleaner. Ya sure I'll make the change to this instead :ok_hand: